src/main/webapp/components/directives/slider/slider.js

```diff
@@ -0,0 +1,92 @@
+(function () {
+  'use strict';
+
+  var module = angular.module('ispn.directives.slider', []);
+
+  module.directive('slider', function () {
+      return {
+        restrict: 'E',
```

Comment:
I don't think it is needed as no ui slider works on div element. See http://refreshless.com/nouislider/examples/ If I set replace to true then slider disappears.